nav2_bt_navigator/behavior_trees/navigate_to_pose_w_replanning_goal_patience_and_recovery.xml

```diff
@@ -0,0 +1,49 @@
+
+<!--
+  This Behavior Tree replans the global path periodically at 1 Hz and it also has
+  recovery actions specific to planning / control as well as general system issues. 
+  This will be continuous if a kinematically valid planner is selected.
+  Additionally it has a feature to wait near the goal to have a new valid plan 
+  if the old plan becomes invalid before trying to drive significantly 
+  around by a length factor.
+-->
+<root main_tree_to_execute="MainTree">
+  <BehaviorTree ID="MainTree">
+    <RecoveryNode number_of_retries="6" name="NavigateRecovery">
+      <PipelineSequence name="NavigateWithReplanning">
+        <RateController hz="1.0">
+          <RecoveryNode number_of_retries="1" name="ComputePathToPose">
+            <ComputePathToPose goal="{goal}" path="{path}" planner_id="GridBased"/>
+            <ClearEntireCostmap name="ClearGlobalCostmap-Context" service_name="global_costmap/clear_entirely_global_costmap"/>
+          </RecoveryNode>
+        </RateController>
+        <ReactiveSequence name="MonitorAndFollowPath">
+          <IsPathLongerOnApproach path="{path}" prox_leng="0.0" length_factor="1.0">
```

Comment:
This is also semantically difficult to read. The `is_` prefix is used for condition nodes, but this is a decorator. I went a little on a rabbit chance to explain why we need to invert this logic but its actually the case that this is correct but the name is deceptive that this is not a condition node, but a decorator. 

I think `Is` needs to be replaced from the node's name, file name, and anywhere else it appears, to just being `PathLongerOnApproach`